spec/rails_helper.rb

```diff
@@ -13,6 +13,10 @@
 require 'support/request_helpers'
 require 'webmock/rspec'
 require 'paper_trail/frameworks/rspec'
+require 'capybara/rails'
```

Comment:
Ha. I dislike Capybara :), but I needed a way to test behaviour around cookies. Personally, I think feature specs are too easy to abuse and can be frustratingly brittle. I think we should write them sparingly, if at all. Yes, definitely a topic for our check-in.